SU-HSDO/suhumsci
# STN-624: Spotlight Color Refinement for Lake Color Pairing

# READY FOR REVIEW

## Summary
Update the background color for Spotlight element for the Lake pairing on the Colorful theme.

## Need Review By (Date)
12/9

## Urgency
low

## Steps to Test
1. In your CLI, run `npm run test` and confirm our tests pass.
2. A [new page](https://sparkbox-sandbox-stage.stanford.edu/qa-spotlight) for testing and QA'ing the Spotlight component has been created on staging. You can either sync with staging and use the [QA Spotlight](http://sparkbox-sandbox.suhumsci.loc/qa/qa-spotlight) page for testing, or create a flexible page and fill it with Spotlight components.
3. The work in this card changes the background color for the Lake color pairing only. You will need to set your theme to `Colorful` and select the `Lake` Color Pairing.
4. Confirm:
- [x] The color pairing of `Lake` uses the background color of `secondary-highlight`
- [x] All other color pairings continue to use the background color of `spotlight`
